kubectl-pg/cmd/util.go

```diff
@@ -0,0 +1,141 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	v1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+const (
+	OperatorName = "postgres-operator"
```

Comment:
When using the helm chart the name of the operator can be different. Then things like `Deployments(namespace).Get(OperatorName...` will not work, unfortunately. However, the name of the label will always be `postgres-operator`. Maybe you can instead filter the deployment by the label `name: postgres-operator` or `app.kubernetes.io/name: postgres-operator` (helm chart convention).

Reply:
WIll search using the label will it work? If so I will change to search using the label. Any corner cases needs to be handled?